sql/wsrep_mysqld.cc

```diff
@@ -2181,6 +2181,12 @@ int wsrep_to_isolation_begin(THD *thd, const char *db_, const char *table_,
   if (!wsrep_thd_is_local(thd))
     return 0;
 
+  if (thd->wsrep_wait_for_parallel_slave_prior_commit())
+  {
+    WSREP_INFO("TOI: wait_for_prior_commit() returned error.");
+    return -1;
```

Comment:
should this be WSREP_WARN instead? This happens only if wait_for_prior_commit() has failed, after all